build_debian.sh

```diff
@@ -211,6 +212,27 @@ sudo cp -f files/sshd/sshd.service $FILESYSTEM_ROOT/lib/systemd/system/ssh.servi
 ## Config sshd
 sudo augtool --autosave "set /files/etc/ssh/sshd_config/UseDNS no" -r $FILESYSTEM_ROOT
 
+## Config monit
+sudo sed -i '
+    s/^# set logfile syslog/set logfile syslog/;
+    s/^\s*set logfile \/var/# set logfile \/var/;
+    s/^# set httpd port/set httpd port/;
```

Comment:
ref: https://mmonit.com/monit/documentation/#MONIT-HTTPD
It enable cli to check status. The access is granted to local connection, so it should be safe.